### PR TITLE
Feature/appointment service

### PR DESCRIPTION
**Pull request by rutaRoze**

### CODE REVIEW

File: appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public ResponseEntity<AppointmentResponse> createAppointment(
+            @Valid @RequestBody AppointmentRequest appointmentRequest) {
+        AppointmentResponse appointmentResponse = appointmentService.saveAppointment(appointmentRequest);
+
+        return new ResponseEntity<>(appointmentResponse, HttpStatus.CREATED);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<AppointmentResponse> getAppointmentById(
+            @Min(1) @PathVariable Long id) {
```
Comment (sukhmeenkaur) [inline appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java:39]: Let me know how this Min works

File: appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        return new ResponseEntity<>(appointmentResponse, HttpStatus.CREATED);
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> deleteAppointmentById(
+            @Min(1) @PathVariable Long id) {
+        appointmentService.deleteAppointmentById(id);
+
+        return ResponseEntity.noContent().build();
```
Comment (sukhmeenkaur) [inline appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java:59]: this should not be "no content'

File: appointment-service/src/main/java/com/roze/appointment_service/dto/response/UserResponse.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserResponse {
+    private Long id;
+    private String name;
+    private String surname;
+    private String email;
+    private String phoneNumber;
+    private Set<RoleName> roleNames;
```
Comment (sukhmeenkaur) [inline appointment-service/src/main/java/com/roze/appointment_service/dto/response/UserResponse.java:21]: Check if you can use list out here

File: appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java (hunk 2 above)
Comment (rutaRoze) [reply appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java:59]: fixed

File: appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java (hunk 1 above)
Comment (rutaRoze) [reply appointment-service/src/main/java/com/roze/appointment_service/controller/AppointmentController.java:39]: The annotated element must be a number whose value must be higher or equal to the specified minimum. null elements are considered valid according to this annotation.

**Pull request merged**